### PR TITLE
add securitywiki migrated infra

### PR DESCRIPTION
**Pull request by cmharlow**
Jira: https://mozilla-hub.atlassian.net/browse/SE-2159

What this PR does:
* migrates & updates SecurityWiki terraform-managed infrastructure from https://github.com/mozilla-it/securitywiki-infra to terraform/applications/securitywiki
* adds two outputs to shared infrastructure to be able to access the cluster & nodegroup security groups (used by security wiki for EFS mount permissions)

### CODE REVIEW

**Review by duallain — APPROVED** [review]
LGTM, seems to match staging, and makes the items we expect.

File: terraform/applications/securitywiki/mysql.tf [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+}
+
+resource "aws_db_subnet_group" "securitywiki_db" {
+  name        = "${var.project}-${var.environment}-rds-subnet-group"
+  description = "${var.project}-${var.environment}-rds-subnet-group"
+  subnet_ids  = flatten([data.terraform_remote_state.vpc.outputs.private_subnets])
+}
+
+resource "aws_db_parameter_group" "securitywiki_db" {
+  name   = "${var.project}-${var.environment}-pg"
```
Comment (bkochendorfer) [inline terraform/applications/securitywiki/mysql.tf:13]: Taking off `pg` as I thought that might mean postgres. Feel free to keep it if I misunderstood.
```suggestion
  name   = "${var.project}-${var.environment}"
```

**Review by bkochendorfer — APPROVED** [review]
one small comment but no big deal!

File: terraform/applications/securitywiki/mysql.tf (hunk 1 above)
Comment (cmharlow) [reply terraform/applications/securitywiki/mysql.tf:13]: yeah i thought that a few times too, makes sense to remove.

**Comment by cmharlow** [comment]
before i forget, updating the references to match @bkochendorfer changes.

**Comment by cmharlow** [comment]
reverting @bkochendorfer recommendation bc it would cause pg turnover and mismatch prod. going easiest route to leave this somewhere i can remember in a week.

**Comment by cmharlow** [comment]
successfully applied.

**Pull request merged**